Roman to Integer/Roman_to_Integer.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+class Solution {
+    func romanToInt(_ s: String) -> Int {
+        let romanToIntDics: [Character:Int] = ["I":1, "V":5, "X":10, "L":50, "C":100, "D":500, "M":1000]
+        var changeS = s.replacingOccurrences(of: "IV", with: "IIII")
+            .replacingOccurrences(of: "IX", with: "VIIII")
+            .replacingOccurrences(of: "XL", with: "XXXX")
+            .replacingOccurrences(of: "XC", with: "LXXXX")
+            .replacingOccurrences(of: "CD", with: "CCCC")
+            .replacingOccurrences(of: "CM", with: "DCCCC")
+        var sum = 0
+        
+        for ch in changeS {
```

Comment:
이 부분 swift 에는 혹시 sum() 이나 map() 같은 iterator 없나요?
있으면 좀 더 코드를 간결하게 만들 수 있을 것 같아요

Reply:
참고해서 찾아보니 map() 고차함수를 사용할 수는 있을 것 같은데, map()의 return이 배열이라 코드가 더 길어지지 않을까 싶네요🧐

Reply:
```suggestion
        var sum = changeS.map({ (ch: Character) -> Int in return romanToIntDics[ch]! }).reduce(0, +)
```
이런식으로 작성할 수 있을 것 같네요

Reply:
Wow.. 되는군요👏